templates/genai-perf-templates/README_template

```diff
@@ -666,8 +684,6 @@ second. For example, 'request_latency:300' or
 'output_token_throughput_per_request:600'. Multiple key:value pairs can be
 provided, separated by spaces. (default: `None`)
 
-</br>
```

Comment:
Did you accidentally delete this line break? It's in the regular README.

Reply:
I'm going to blame this on my cat 🐱